### PR TITLE
static_line_chute: add "All Groups" to keep groups together

### PR DESCRIPTION
**Pull request by freghar**
Signed-off-by: freghar <freghar@dummy.tld>

### CODE REVIEW

File: addons/static_line_chute/fn_calcJumpDelay.sqf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+private _spd = abs speed _this;
+switch true do {
+    case (_spd > 200): {0.5};
+    case (_spd > 150): {1};
+    case (_spd > 100): {2};
+    case (_spd > 50):  {4};
+    default            {6};
```
Comment (lastmikoi) [inline addons/static_line_chute/fn_calcJumpDelay.sqf:7]: I suppose wolframalpha's suggestion for a fit is a tad too overkill to implement :D:

`0.000114286 x^2 - 0.0508571 x + 6.07143`

File: addons/static_line_chute/fn_getPassengers.sqf [hunk 2]
```diff
@@ -9,21 +9,30 @@
...
-    /* limit to one group, if specified */
-    && {isNull _group || group _x == _group}
-    /* no ACE prisoners or ACE unconscious units */
-    && !(_x getVariable ["ace_captives_isHandcuffed", false])
-    && !(_x getVariable ["ACE_isUnconscious", false])
+/*
+ * run this atomically
+ *
+ * it's a good idea in general, but it also works around an engine bug
+ * in select{} which randomly returns undefined _x in scheduled env
```
Comment (lastmikoi) [inline addons/static_line_chute/fn_getPassengers.sqf:17]: Well shite, good job troubleshooting that one...

**Review by lastmikoi — APPROVED** [review]
LGTM, needs a single review from a @CntoDev/ssgt  to get merged.

File: addons/static_line_chute/fn_calcJumpDelay.sqf (hunk 1 above)
Comment (freghar) [reply addons/static_line_chute/fn_calcJumpDelay.sqf:7]: Didn't want to mix two things at once and figured we can optimize this later.

The correct solution would be figuring out a safe distance between paradropped units (ie. 21m) and calculating delay based on speed/velocity (ie. 150km/h, ~42m/s): `v = dx / dt`, that is `t = dx/v`, `t = 21/42 = 0.5s`.

So either the `speed` command (in km/h) for forward velocity, or the `velocity` command (in m/s) and selecting from its array or doing trigonometry to account for sideways velocity as well.

**Review by CNTODachi — APPROVED** [review]

**Pull request merged**